core/src/main/resources/defaults/blueprints/7.2.11/cdp-data-engineering-ha.bp

```diff
@@ -494,15 +494,16 @@
           "hive-HIVESERVER2-BASE",
           "hms-GATEWAY-BASE",
           "hms-HIVEMETASTORE-BASE",
-          "oozie-OOZIE_SERVER-BASE",
           "livy-GATEWAY-BASE",
           "spark_on_yarn-GATEWAY-BASE",
           "tez-GATEWAY-BASE",
           "yarn-RESOURCEMANAGER-BASE",
           "hdfs-JOURNALNODE-BASE",
           "zookeeper-SERVER-BASE",
           "yarn-GATEWAY-BASE",
-          "sqoop-SQOOP_CLIENT-GATEWAY-BASE"
+          "sqoop-SQOOP_CLIENT-GATEWAY-BASE",
+          "hue-HUE_LOAD_BALANCER-BASE",
```

Comment:
Yes, but a slightly different scenario. We do not use Knox for internal communications. Here the motivation is Hue and Oozie should not be in the same host group. Since we are using network load-balancers, they can not communicate with themselves.

Reply:
Ah, Hue needs to communicate with Ooozie or vice versa, network load balancers don't permit routing back to the node that initiated the connection.

Thanks!